scripts/target_lib.lua

```diff
@@ -44,6 +45,7 @@ local default_options = {
 	aliases = {},
 	manifest_aliases = {},
 	packages = {},
+	featureset = false,
```

Comment:
You could just define `featureset = "default"` here and undo all other changes in this file. Rather than adding a separate top-level function `featureset` for the defaults, this would be handled by the existing `defaults` function.